tests/console/yast2_i.pm

```diff
@@ -51,16 +52,15 @@ sub run {
     }
 
     # check required automatic updates, it is not scanned on sle12 codestream
-    if (is_sle('>=15-sp1') || is_tumbleweed || is_leap('>=15.1') || is_jeos) {
+    if (is_sle('>=15-sp1') || is_tumbleweed || is_leap('>=15.1')) {
```

Comment:
Why this  change is required? Dosen't look to be related to the idea of the PR...

Reply:
Well, yes and no. I did not want to keep yast2_i.pm blocked. Because the previous condition was always true for any kind of jeos, which is incorrect by default. As long as JeOS has many versions which are still supported, I had to somehow determine its version and the tradition is_{distri} works well.

Reply:
issue:

http://eris.suse.cz/tests/13649#step/yast2_i/13

Reply:
Ah, now I recall. Maybe it's worth putting it in separate commit in this PR. Thanks for the explanation!

Reply:
Excellent! I will put it a separate commit, thanks!